rockcraft/extensions/_utils.py

```diff
@@ -116,3 +117,8 @@ def _remove_list_duplicates(seq: list[str]) -> list[str]:
             deduped.append(item)
 
     return deduped
+
+
+def find_ubuntu_base_python_version(base: str) -> str | None:
+    """Find the Python version for the given base."""
+    return UBUNTU_PYTHON_VERSION_MAP.get(base, None)
```

Comment:
Perhaps just make it an error if an unknown base is requested? So that you don't have to do a None check everywhere